lib/apiservers/engine/backends/backends.go

```diff
@@ -253,7 +257,7 @@ func createImageStore() error {
 }
 
 func InsecureRegistries() []string {
-	registries := make([]string, len(insecureRegistries))
+	registries := []string{}
```

Comment:
This is incorrect fix. Initial line had a bug not providing size only.
Correct fix is this:
```registries := make([]string, 0, len(insecureRegistries))```